### PR TITLE
 Insert pci review 

### PR DESCRIPTION
**Pull request by simorjay**
Link to the PCI review was added to the index

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/compliance/pci-dss/index.md](https://github.com/simorjay/architecture-center/blob/master/docs/compliance/pci-dss/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/compliance/pci-dss/index?branch=pr-en-us-242) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C3%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201710032257534075-242%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=n8Fuzmq%2BEY5eMWOC9Fhjq5nhXe9KYQADdLlwGXjrpNE%3D&st=2017-10-03T22%3A54%3A28Z&se=2017-10-04T23%3A04%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/compliance/pci-dss/index.md [hunk 1]
```diff
@@ -352,6 +352,10 @@ For more information, see the [PCI Blueprint Threat Model](https://aka.ms/pciblu
 
 Customers are responsible for retaining a copy of the [Responsibility Summary Matrix](https://aka.ms/pciblueprintcrm32), which outlines the PCI DSS requirements that are the responsibility of the customer and those which are the responsibility of Microsoft Azure.
 
+## PCI Compliance Review 
+
+The solution was reviewed by Coalfire systems, Inc. (PCI-DSS Qualified Security Assessors). The [PCI Compliance Review and guidance for implementation](https://aka.ms/pciblueprintprocessingoverview) provides an auditors review of the solution, and considerations to transform the blueprint, to production ready.
```
Comment (MikeWasson) [inline docs/compliance/pci-dss/index.md]: auditors => auditor's

Comment (MikeWasson) [inline docs/compliance/pci-dss/index.md]: Suggestion:

"considerations to transform the blueprint, to production ready" => "considerations for transforming the blueprint to a production-ready deployment."

**Comment by MikeWasson** [comment]
@simorjay see review comments

**Comment by MikeWasson** [comment]
Replacing this PR with #245

File: docs/compliance/pci-dss/index.md (hunk 1 above)
Comment (simorjay) [reply docs/compliance/pci-dss/index.md]: sounds good. I'll update and pull

**Pull request closed without merging**